### PR TITLE
Restaurant list screen

### PR DESCRIPTION
**Pull request by DonatasPuc**
Added first iteration of restaurant list screen. For testing purposes it is also filled with some generated data.

image

### CODE REVIEW

File: wasted-app/RestaurantListControl.cs [hunk 1]
```diff
@@ -25,6 +27,37 @@ public static RestaurantListControl Instance
...
+
+            AddAllRestaurants();
+        }
+
+        public void AddAllRestaurants()
+        {
+            ServicesController controller = new ServicesController();
+            var restaurants = controller.RestaurantService.GetAllRestaurants();
+
+            foreach (var r in restaurants)
```
Comment (audrius-savickas) [inline wasted-app/RestaurantListControl.cs:39]: maybe not `var r` but `Restaurant restaurant`?

File: wasted-app/RestaurntControl.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace wasted_app
+{
+    public partial class RestaurntControl : UserControl
```
Comment (JesusGonzalezA) [inline wasted-app/RestaurntControl.cs:13]: restaurant io restaurnt

File: wasted-app/RestaurantListControl.cs [hunk 3]
```diff
@@ -25,6 +27,30 @@ public static RestaurantListControl Instance
         public RestaurantListControl()
         {
             InitializeComponent();
+
+            AddAllRestaurants();
+        }
+
+        public void AddAllRestaurants()
+        {
+            ServicesController controller = new ServicesController();
```
Comment (JesusGonzalezA) [inline wasted-app/RestaurantListControl.cs:36]: Use singleton io of instansciate another one

Comment (JesusGonzalezA) [reply wasted-app/RestaurantListControl.cs:36]: ServicesController.Instance

**Review by JesusGonzalezA — CHANGES_REQUESTED** [review]

File: wasted-app/RestaurntControl.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        public RestaurntControl()
```
Comment (audrius-savickas) [inline wasted-app/RestaurntControl.cs:15]: typo

File: wasted-app/RestaurantListControl.cs (hunk 1 above)
Comment (DonatasPuc) [reply wasted-app/RestaurantListControl.cs:39]: I think it's fine to use it in this situation, no need to be explicit here.

File: wasted-app/RestaurntControl.cs (hunk 2 above)
Comment (DonatasPuc) [reply wasted-app/RestaurntControl.cs:13]: Fixed

**Pull request merged**